010_experiments/010_experiments.tex

```diff
@@ -1,8 +1,383 @@
+
+We also assert that this graph details the implied benefit of signing treaties: agents seem to value treaties as a means to fairer food distribution, wherein signing facilitates them receiving a greater proportion of food than they would otherwise receive without a treaty.
+
+\begin{figure}[H] %Deaths As Food Scarcity Decreases with different agent combos added
+    \centering
+    \begin{tikzpicture}
+        \begin{axis}[
+            width=0.8\textwidth,
+            % axis y line*=left,
+            ymin=0,
+            xmin=0,
+            xlabel=Available Food Per Agent,
+            ylabel=Total Number of Agent Deaths,
+            ymajorgrids=true,
+            xmajorgrids=true,
+            grid style=dashed,
+        ]
+        \addplot[mark=*,green]
+            coordinates{
+                (2, 484)
+                (5, 250)
+                (10, 47)
+                (20, 6)
+            }; \label{leg:scarcity00}
+        \addplot[mark=*,red]
+            coordinates{
+                (2, 618)
+                (5, 240)
+                (10, 155)
+                (20, 4)
+            }; \label{leg:scarcity03}
+        \addplot[mark=*,blue]
+            coordinates{
+                (2, 618)
+                (5, 426)
+                (10, 88)
+                (20, 4)
+            }; \label{leg:scarcity30}
+        \addplot[mark=*,orange]
+            coordinates{
+                (2, 741)
+                (5, 529)
+                (10, 222)
+                (20, 12)
+            }; \label{leg:scarcity33}
+        \addlegendimage{/pgfplots/refstyle=leg:scarcity00}\addlegendentry{0,0}
+        \addlegendimage{/pgfplots/refstyle=leg:scarcity03}\addlegendentry{0,3}
+        \addlegendimage{/pgfplots/refstyle=leg:scarcity30}\addlegendentry{3,0}
+        \addlegendimage{/pgfplots/refstyle=leg:scarcity33}\addlegendentry{3,3}
+        \end{axis}
+        \end{tikzpicture}
+    \caption{The total number of deaths in systems with different combinations of selfish and random agents added as food per agent increases. The number of each disruptive agent added to each system is show in the legend in the form (number of selfish agents, number of random agents)}
+    \label{fig:team1-robustness-food-scarcity-disruptive-agents}
+\end{figure}
+
+\Cref{fig:team1-robustness-food-scarcity-disruptive-agents} shows how the number of deaths is affected when different disruptive agents are added. It shows that, as expected, the number of deaths is higher when disruptive agents are present. However, the effect of selfish agents and random agents seem to be similar which suggests that it it not the amount of food being taken by the agents that is disruptive, rather it is the fact that they do not communicate that causes a higher death rate. This makes it harder to self organise and makes it much harder to survive in the tower. 
```

Comment:
self organise or self-organise? I've seen the latter around quite often

Reply:
will change to the latter